src/components/TopBar.tsx

```diff
@@ -17,7 +17,7 @@ const Bar = styled('div')({
   display: 'flex',
   flexDirection: 'row',
   alignItems: 'center',
-  justifyContent: 'end',
+  justifyContent: 'space-between',
```

Comment:
I think this change is the reason the content is skewed to the right when a button isn't present